File: MORM.Java/src/MORM/Java/Models/Venda/Caixacont.java
package MORM.Java.Models.Venda;

import java.util.Date;
import MORM.Java.Classes.*;
import MORM.Java.Classes.Mapping.*;

@TabelaAttribute(Nome = "CAIXACONT")
public class Caixacont extends CollectionItem
{
    @CampoAttribute(Campo = "ID_CAIXA", Tipo = CampoTipo.Key)
    public Integer Id_Caixa;
    @CampoAttribute(Campo = "ID_HISTREL", Tipo = CampoTipo.Key)
    public Integer Id_Histrel;
    @CampoAttribute(Campo = "U_VERSION", Tipo = CampoTipo.Nul)
    public String U_Version;
    @CampoAttribute(Campo = "CD_OPERADOR", Tipo = CampoTipo.Req)
    public Integer Cd_Operador;
    @CampoAttribute(Campo = "DT_CADASTRO", Tipo = CampoTipo.Req)
    public Date Dt_Cadastro;
    @CampoAttribute(Campo = "VL_CONTADO", Tipo = CampoTipo.Req)
    public Float Vl_Contado;
    @CampoAttribute(Campo = "VL_SISTEMA", Tipo = CampoTipo.Req)
    public Float Vl_Sistema;
    @CampoAttribute(Campo = "VL_RETIRADA", Tipo = CampoTipo.Req)
    public Float Vl_Retirada;
    @CampoAttribute(Campo = "VL_SUPRIMENTO", Tipo = CampoTipo.Req)
    public Float Vl_Suprimento;
    @CampoAttribute(Campo = "VL_DIFERENCA", Tipo = CampoTipo.Req)
    public Float Vl_Diferenca;
}

File: MORM.CSharp/MORM.Dominio/Models/Venda/Caixacont.cs
using MORM.Util.Atributos;
using MORM.Util.Classes;
using System;

namespace MORM.CSharp.Models
{
    [Tabela("CAIXACONT")]
    public class Caixacont : CollectionItem
    {
        [Campo("ID_CAIXA", CampoTipo.Key)]
        public int Id_Caixa { get; set; }
        [Campo("ID_HISTREL", CampoTipo.Key)]
        public int Id_Histrel { get; set; }
        [Campo("U_VERSION", CampoTipo.Nul)]
        public string U_Version { get; set; }
        [Campo("CD_OPERADOR", CampoTipo.Req)]
        public int Cd_Operador { get; set; }
        [Campo("DT_CADASTRO", CampoTipo.Req)]
        public DateTime Dt_Cadastro { get; set; }
        [Campo("VL_CONTADO", CampoTipo.Req)]
        public double Vl_Contado { get; set; }
        [Campo("VL_SISTEMA", CampoTipo.Req)]
        public double Vl_Sistema { get; set; }
        [Campo("VL_RETIRADA", CampoTipo.Req)]
        public double Vl_Retirada { get; set; }
        [Campo("VL_SUPRIMENTO", CampoTipo.Req)]
        public double Vl_Suprimento { get; set; }
        [Campo("VL_DIFERENCA", CampoTipo.Req)]
        public double Vl_Diferenca { get; set; }
    }
}